qemu/tests/hotplug_block_resize.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import re
+import json
+
+from virttest import error_context
+from virttest import utils_disk
+from virttest import utils_numeric
+from virttest import utils_test
+from virttest import storage
+from virttest import qemu_storage
+from virttest import data_dir
+
+from provider.plug_block_devices import SerialPlugManger
+from provider.storage_benchmark import generate_instance
```

Comment:
There is no `provider.plug_block_devices` for `provider/block_devices_plug.py`, so it occurred error as` ERROR: No module named 'provider.plug_block_devices'`
And a typo for SerialPlugManger, it should be "Manager"